### PR TITLE
Add information to actions page

### PR DESCRIPTION
**Pull request by shanear**
This requires ClaimEstablishment to exist in order to work, and renames a column on ClaimEstablishment, so I'll be deploying this weekend.

connects #1144

### CODE REVIEW

File: spec/feature/establish_claim_spec.rb [hunk 1]
```diff
@@ -64,461 +64,477 @@
   context "As a caseworker" do
     let!(:current_user) { User.authenticate!(roles: ["Establish Claim"]) }
 
-    context "Establishing a claim" do
-      let!(:task) do
-        Generators::EstablishClaim.create(appeal_id: appeal.id, aasm_state: "unassigned")
-      end
+    let!(:task) do
```
Comment (shanear) [inline spec/feature/establish_claim_spec.rb]: most of the changes here come from removing an unnecessary context.

Ill try and highlight the actual additons

File: spec/feature/establish_claim_spec.rb [hunk 2]
```diff
@@ -64,461 +64,477 @@
...
-        expect(task.reload.user).to eq(current_user)
-        expect(task).to be_started
+      # Validate the unassigned task was assigned to me
+      expect(page).to have_current_path("/dispatch/establish-claim/#{task.id}")
+      expect(task.reload.user).to eq(current_user)
+      expect(task).to be_started
 
-        visit "/dispatch/establish-claim"
-        click_on "Establish next claim"
+      # Validate that a Claim Establishment object was created
```
Comment (shanear) [inline spec/feature/establish_claim_spec.rb]: Added this

File: spec/feature/establish_claim_spec.rb [hunk 3]
```diff
@@ -64,461 +64,477 @@
...
-          find_label_for("dicDeathOrAccruedBenefitsUnitedStates").click
+        click_on "Route claim"
 
-          click_on "Route claim"
+        expect(find_field("Station of Jurisdiction").value).to eq("351 - Muskogee, OK")
 
-          expect(find_field("Station of Jurisdiction").value).to eq("351 - Muskogee, OK")
+        click_on "Create End Product"
+        find_label_for("confirmNote").click
+        click_on "Finish routing claim"
```
Comment (shanear) [inline spec/feature/establish_claim_spec.rb]: added lines 229-231

File: spec/feature/establish_claim_spec.rb [hunk 4]
```diff
@@ -64,461 +64,477 @@
...
 
-          click_on "Finish routing claim"
+        # Ensure that the user stays on the note page on a refresh
+        visit "/dispatch/establish-claim/#{task.id}"
+
+        expect(find(".cf-app-segment > h2")).to have_content("Route Claim")
+        find_label_for("confirmNote").click
+
+        click_on "Finish routing claim"
+
```
Comment (shanear) [inline spec/feature/establish_claim_spec.rb]: Added lines 445-448

File: app/models/appeal.rb [hunk 5]
```diff
@@ -25,21 +25,36 @@ class MultipleDecisionError < StandardError; end
...
+    rice_compliance: "Rice Compliance",
+    spina_bifida: "Spina Bifida",
+    us_territory_claim_american_samoa_guam_northern_mariana_isla: "U.S. Territory claim - American Samoa, Guam, Northern Mariana Islands (Rota, Saipan & Tinian)",
+    us_territory_claim_philippines: "U.S. Territory claim - Philippines",
+    us_territory_claim_puerto_rico_and_virgin_islands: "U.S. Territory claim - Puerto Rico and Virgin Islands",
+    vamc: "VAMC",
+    vocational_rehab: "Vocational Rehab",
+    waiver_of_overpayment: "Waiver of Overpayment"
+  }.freeze
+  # rubocop:enable Metrics/LineLength
```
Comment (sharonwarner) [inline app/models/appeal.rb]: For the regional offices, we send the info to the React code as a prop. Should we move all of the special issue constants to Rails and send it the same way? It seems a little risky to assume these special issues will remain in sync between Rails and React.

Comment (shanear) [inline app/models/appeal.rb]: yeah, we need to solve that. I'll create a tech improvement for that

**Pull request merged**